auslib/db.py

```diff
@@ -1914,6 +1914,9 @@ def setupChangeMonitors(self, relayhost, port, username, password, to_addr, from
         self.rules.onInsert = bleeter
         self.rules.onUpdate = bleeter
         self.rules.onDelete = bleeter
+        self.rules.scheduled_changes.onInsert = bleeter
+        self.rules.scheduled_changes.onUpdate = bleeter
+        self.rules.scheduled_changes.onDelete = bleeter
```

Comment:
Almost ready for a couple of loops here ? eg for table in (self.rules, ...):  for attr ('onInsert', ...)  setattr(table, attr, bleeter)